module/solutions/validator/v1.js

```diff
@@ -24,6 +24,13 @@ module.exports = (req) => {
             req.checkParams('_id').exists().withMessage("required observation solution id");
             req.checkQuery('appName').exists().withMessage("required app name");
         },
+        addEntities : function () {
+            req.checkParams("_id").exists().withMessage("Required solution id");
+            req.checkBody("entities").exists().withMessage("Required entities data");
+        },
+        list : function () {
+            req.checkBody("solutionIds").exists().withMessage("Required solution external ids");
```

Comment:
@aman-tunerlabs - We can check if its array and all are mongoId as well

Reply:
@aks30 we are cehcking with externalId not mongoId